react/states_and_effects/more_on_state.md

```diff
@@ -169,34 +169,34 @@ There are native HTML elements that maintain their own internal state. The `inpu
 
 ~~~jsx
 function CustomInput() {
-   const [value, setValue] = useState('');
-   
-   return (
-      <input
-         type="text"
-         value={value}
-         onChange={(event) => setValue(event.target.value)}
-      />
-   );
+  const [value, setValue] = useState('');
+
+  return (
+    <input
+        type="text"
+        value={value}
+        onChange={(event) => setValue(event.target.value)}
+    />
+  );
 }
 ~~~
 
-Instead of letting the `input` maintain its own state, we define our own state using the `useState` hook. We then set the `value` prop of the `input` to the state variable and update the state variable on every `onChange` event. Now, every time the user types something in the input, React will ensure you have the latest comment/review/post (whatever the user was typing) in `value`. 
+Instead of letting the `input` maintain its own state, we define our own state using the `useState` hook. We then set the `value` prop of the `input` to the state variable and update the state variable on every `onChange` event. Now, every time the user types something in the input, React will ensure you have the latest comment/review/post (whatever the user was typing) in `value`.
 
 This pattern is extremely useful wherever you need user input i.e. typing in a textbox, toggling a checkbox etc. Contrarily, yes, the `input` element can be left uncontrolled and access its value through some other method. You don't need to worry about it yet as it is covered later on in the course. For now, control your components!
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Read the following articles from the React documentation: 
-   - [State as a Snapshot](https://react.dev/learn/state-as-a-snapshot)
-   - [Choosing the State Structure](https://react.dev/learn/choosing-the-state-structure)
-   - [Sharing State Between Components](https://react.dev/learn/sharing-state-between-components)
+1. Read the following articles from the React documentation:
+  - [State as a Snapshot](https://react.dev/learn/state-as-a-snapshot)
+  - [Choosing the State Structure](https://react.dev/learn/choosing-the-state-structure)
+  - [Sharing State Between Components](https://react.dev/learn/sharing-state-between-components)
 
 2. Update the `Person` component we've been using above.
-   - Add two separate input fields for first name and last name. The updated full name should be displayed on every keystroke on either of the two input fields. 
-   - There are many ways you can do this. Keep in mind what you've learned in this lesson while coding it out.
+  - Add two separate input fields for first name and last name. The updated full name should be displayed on every keystroke on either of the two input fields.
+  - There are many ways you can do this. Keep in mind what you've learned in this lesson while coding it out.
```

Comment:
These will need to be reverted back to 3 spaces, according to our style guide.
> a 3 space indent when nesting inside a numbered list
https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#nested-lists

Reply:
That's my bad! I thought I account for that but must have slipped through...now reverted.